app/Http/Controllers/Api/Address/AddressController.php

```diff
@@ -46,6 +48,11 @@ public function store(Request $request)
 
         $this->authorize('create', $address);
 
+        $contact = Contact::find($data['contactId']);
+        if($contact){
+            $contactAddresAllowed = $this->checkDoubleAddressAllowed($contact, $address);
```

Comment:
`contactAddresAllowed` address is met dubbel s aan het eind

Reply:
Variable name changed (address with double s)